### PR TITLE
improperly working scripts for 1.19.1 are moved to deprecated + test_…

### PR DESCRIPTION
**Pull request by CICS-Oleg**
…horizon is fixed and got rid of MalmoPython references

### CODE REVIEW

**Review by DaddyWesker — APPROVED** [review]

File: experiments/test_horizon.py [hunk 1]
```diff
@@ -209,10 +209,6 @@ def show_horizon(rob, height=64, horizon_dist=200):
 def main():
     mc, rob = start_mission()
     mc.safeStart()
-    MalmoPython.setLogging("log.txt", MalmoPython.LoggingSeverityLevel.LOG_FINE)
-    MalmoPython.setLoggingComponent(MalmoPython.LoggingComponent.LOG_VIDEO, True)
-    MalmoPython.setLoggingComponent(MalmoPython.LoggingComponent.LOG_TCP, True)
-    MalmoPython.setLoggingComponent(MalmoPython.LoggingComponent.LOG_AGENTHOST, True)
```
Comment (Necr0x0Der) [inline experiments/test_horizon.py:215]: Is there an alternative to MalmoPython logging?

**Comment by Necr0x0Der** [comment]
> improperly working scripts for 1.19.1 are moved to deprecated

I see only one commit for `test_horizon` in this PR. Nothing is moved.

File: experiments/test_horizon.py (hunk 1 above)
Comment (CICS-Oleg) [reply experiments/test_horizon.py:215]: There is no such interface in python version of the library. It has logging mechanism throughtout standart logger, but I don't know if it implements features similar to MalmoPython.

**Comment by CICS-Oleg** [comment]
I added logger from _init_.py.

**Comment by CICS-Oleg** [comment]
> Nothing is moved.
test_colourmap.py and test_skills.py have been moved to deprecated

File: experiments/test_horizon.py (hunk 1 above)
Comment (CICS-Oleg) [reply experiments/test_horizon.py:215]: I've added logger from `__init__.py` to test_horizon.py

**Review by Necr0x0Der — APPROVED** [review]

**Pull request merged**